books/book.html

```diff
@@ -12,19 +16,94 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
-    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
-    <title>Book Details</title>
+    <!-- Styling -->
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="book.css" />
+    <title>Book Description :: CWB Books for all!</title>
   </head>
   <body>
     <header>
       <div class="container">
-        <h1><a href="../index.html"> Books for all!</a></h1>
+        <div class="logo">
+          <h1><a href="../index.html">Books for all!</a></h1>
+          <p>A Bookshop using Vanilla JavaScript</p>
+        </div>
+        <div id="login-box">
+          <div id="user-box">
+            <i class="fas fa-user" id="user-icon"></i>
+            <span id="user-show"></span>
+          </div>
+          <button id="btn-Form">
+            Login <i class="fas fa-sign-in-alt"></i>
+          </button>
+        </div>
+
+        <!-- start User Form Popup -->
+        <div class="form-popup" id="form-popup">
+          <form class="form-content" id="login-form">
+            <div class="avatar-container">
+              <span id="popup-close-x"><i class="fas fa-times"></i></span>
+              <img id="avatar" src="" alt="" />
+            </div>
+
+            <label for="uname">Username</label>
+            <input
+              type="text"
+              placeholder="Enter Username"
+              id="uname"
+              required
+            />
+
+            <label for="password">Password</label>
```

Comment:
Wrong id of the input element. As I said in a previous comment, I would like to see full names so keep this as is but change the id of the input element